src/templates/finops-hub/schema/README.md

```diff
@@ -0,0 +1,9 @@
+# 📦 FinOps hub schema tranforms
+
+These transforms are used to convert incoming cost management data into the target schema.
```

Comment:
Either "cost data" or "Cost Management data". Either is fine.
```suggestion
These transforms are used to convert incoming cost data into the target schema.
```

Reply:
Done